torch/ao/quantization/_pt2e/quantizer/quantizer.py

```diff
@@ -24,8 +29,14 @@
     torch.per_channel_affine_float_qparams,
 ]
 
+class QuantizationSpecBase(ABC):
+    """ Base class for all types of quantization specs, this is introduced
+    to keep the typing for quantization annotation API simple
```

Comment:
I feel this comment is a bit too implementation oriented. Maybe it should be something more user oriented like "Base quantization spec that allows users to specify how to quantize a tensor"

Reply:
OK sounds good